README.md

```diff
@@ -50,36 +50,35 @@ The steps above are performed in [`ExtractCCLinks.py`][ex_cc_links].
 various API ETL jobs which pull and process data from a number of open APIs on
 the internet.
 
-### Daily API Workflows
+### API Workflows
+
+Our API-based workflows run at different schedules: some daily, others monthly. Please consider the which to use whenever a new DAG is written, and add your new script to one of these schedules.
```

Comment:
```suggestion
Our API-based workflows run at different schedules: some daily, others monthly. Please consider which to use whenever a new DAG is written, and add your new script to one of these schedules.
```
Seems like this might be accidental